ansible/roles/mcx/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+##############################################################################################
+### playbook to deploy the mcx switch
+### Use this playbook to deploy the console configurations of mcx switch in SONiC testbed
+################################################################################################
+- name: Gathering lab graph facts about the device
+  conn_graph_facts: host={{ inventory_hostname }}
+  delegate_to: localhost
+  tags: always
+
+- name: Check that OS is SONiC
+  fail: msg="Cannot confirm OS is SONiC"
+  when: os is not defined or os != "sonic"
+
+- name: Set HwSku
+  set_fact:
+    HwSku: "{{ device_info[inventory_hostname][\"HwSku\"] }}"
+
+- name: Check that HwSku is Nokia-7215 or Celestica-E1031-T48S4
+  fail: msg="Device HwSku is not Nokia-7215 or Celestica-E1031-T48S4"
+  when: HwSku != "Nokia-7215" and HwSku != "Celestica-E1031-T48S4"
+
+- name: Prepare mgmt device admin login info
+  set_fact: ansible_ssh_user={{ mgmt_admin_user }} ansible_ssh_pass={{ mgmt_admin_password }}
+
+- name: Generate DEVICE_METADATA
+  shell: sonic-cfggen -H -k {{ HwSku }} --preset empty | sed '1d;$d'
+  register: DEVICE_METADATA
+
+- name: Backup config_db.json
+  shell: cp /etc/sonic/config_db.json /etc/sonic/config_db.json.bak
+
+- name: Move json config to console server
+  template:
+    src: console_config.j2
+    dest: /etc/sonic/config_db.json
```

Comment:
How about use `console_config.j2` generate the additional config first, then use `-j` to import it while `sonic-cfggen`? (Just like generate SONiC fanout config_db.json)